### DISCUSSION TITLE
Font Awesome

### DISCUSSION THREAD

**Discussion by inLIMITit** [slabbi/uddeIM#92] [Ideas]
Hello,

is it possible to replace the icons with more modern ones, for example with the FontAwesome font installed by default in Joomla?

**Comment by jojo12** [comment]
Perhaps you can try and then share

**Comment by inLIMITit** [comment]
If I had even halfway good programming skills, I probably would have done it - but unfortunately that's not the case.

**Comment by jojo12** [comment]
When you study fa icons then you will quickly see, it is so difficult to find adequat icons. Proposals are welcome too!

**Comment by joomod** [comment]
I'm not quite shure why you call FA icons modern looking?
scrolling through the icon list on FA shows many nice but mostly "pro" icons.
If you gave me a **list** which icons from FA I should choose,
I may look if it possible to integrate this as an option.

**Reply by inLIMITit** [reply]
The main thing that bothered me was the old-looking design of the icons. So it seemed obvious to use FontAwesome. But if that's too complicated to implement, I can always look for more modern icons as an alternative.

**Comment by joomod** [comment]
the best way to fit your needs is just to place your new icons in the **images folder** 
of your selected template, e.g. all the menu icons are menu_*xxx.gif

**Comment by jojo12** [comment]
What's strange: a lot of icons, gifs or pngs are blind! Why?

**Comment by joomod** [comment]
what do you mean with blind?
(may be the new templates, there are transparent placeholders, and the icon is placed as background)

**Comment by jojo12** [comment]
When I open the images, there seems to be nothing. Are these placeholders? And when can I see the images?
(I will try to use other images as play around)

**Comment by joomod** [comment]
that's the reason, why I have set the module icons to load from the default template.
some of the new templates uses transparent placeholders (menu_xxx.gif) in the direct img src,
and then define background image for this (menu_xxx.png) in the depending css.
just look in your template image folder, there are menu-**gif**s and menu-**png**s

**Comment by joomod** [comment]
for the uddeim_mailbox module I added fontawesome icons as option

**Comment by jojo12** [comment]
wow, you're brilliant!
